aws.yaml

```diff
@@ -1,3 +1,49 @@
+- active: false
+  apps:
+    - app: chart-operator
+      version: 0.11.4
+    - app: coredns
+      version: 1.1.3
+      componentVersion: 1.6.5
```

Comment:
Let's also order the version fields. I think that is tidier.

Reply:
I put it like deliberately, but let's keep in order as you said.